### ISSUE TITLE
Wrong URL in Warning

### ISSUE THREAD

**Issue by LLCampos** [biocommons/eutils#141]
I got this while experimenting the package:

```
WARNING:eutils.client:NCBI found 915 results, but we truncated the reply at 250 results; see https://bitbucket.org/biocommons/eutils/issues/124/
```

That's an old url, and I guess it should be updated? :)

**Comment by reece** [comment]
Yes, all links from bitbucket are broken. I'll update them. Thanks for the reminder.

**Comment by reece** [comment]
Fixed in 0.2.2:
- https://travis-ci.org/biocommons/eutils/builds/182892675
- https://pypi.python.org/pypi/eutils/0.2.2
